tests/Unit/NumericalAlgorithms/Interpolation/Test_RegularGridInterpolant.py

```diff
@@ -0,0 +1,148 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import unittest
+from spectre import Spectral
+from spectre import DataStructures
+from spectre import Interpolation
+from numpy.polynomial.legendre import Legendre
+import numpy as np
+
+
+class TestRegularGrid(unittest.TestCase):
+
+    #random polynomial functions of low order for exact interpolation
+    def polynomial1D(self, points):
+        return points**2 + points + 1
+
+    def polynomial2D(self, points):
+        result = np.empty(len(points))
+        for i, p in enumerate(points):
+            result[i] = 2 * p[0]**2 + p[1]**2 + p[1] + p[0] + 2
+        return result
+
+    def polynomial3D(self, points):
+        result = np.empty(len(points))
+        for i, p in enumerate(points):
+            result[i] = 3 * p[0] ** 2 + 2 * p[1] ** 2 + p[2] ** 2 + 2 * p[1] + \
```

Comment:
You can break math (or just about anything) over multiple lines in Python by wrapping the expression in parens `()`